ARTIFACT_INVENTORY.md

```diff
@@ -28,13 +28,13 @@ registry/repository/image_name:version
 | Metrics | Ghostunnel | docker.io/ghostunnel/ghostunnel:v1.7.1|
 | Metrics | Grafana | docker.io/grafana/grafana:10.0.3|
 | Metrics | Admission Webhook | registry.k8s.io/ingress-nginx/kube-webhook-certgen:v20221220-controller-v1.5.1-58-g787ea74b6|
-| Metrics | Kube State Metrics | registry.k8s.io/kube-state-metrics/kube-state-metrics:v2.9.2|
-| Metrics | Node Exporter | quay.io/prometheus/node-exporter:v1.6.1|
-| Metrics | Prometheus | quay.io/prometheus/prometheus:v2.46.0|
-| Metrics | Prometheus Operator | quay.io/prometheus-operator/prometheus-operator:v0.67.1|
-| Metrics | Configuration Reloader (Alertmanager, Prometheus) | quay.io/prometheus-operator/prometheus-config-reloader:v0.67.1|
-| Metrics | Prometheus Pushgateway | docker.io/prom/pushgateway:v1.6.0|
-| Metrics | Auto-load Sidecars (Grafana) | quay.io/kiwigrid/k8s-sidecar:1.25.0|
+| Metrics | Kube State Metrics | registry.k8s.io/kube-state-metrics/kube-state-metrics:v2.10.0|
+| Metrics | Node Exporter | quay.io/prometheus/node-exporter:v1.7.0|
+| Metrics | Prometheus | quay.io/prometheus/prometheus:v2.47.1|
+| Metrics | Prometheus Operator | quay.io/prometheus-operator/prometheus-operator:v0.69.1|
+| Metrics | Configuration Reloader (Alertmanager, Prometheus) | quay.io/prometheus-operator/prometheus-config-reloader:v0.69.1|
+| Metrics | Prometheus Pushgateway | docker.io/prom/pushgateway:v1.6.2|
```

Comment:
@cumcke Just noticed that the container image in my test cluster is from a different registry/repository  now: moved from "prom" to "quay.io/prometheus".  Looks like there was a change: https://github.com/prometheus-community/helm-charts/commit/0a8cf344765d92bf97c5241d4d9a812ce0b3e5e6